src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldExtendedLookup.class.php

```diff
@@ -152,6 +152,68 @@ protected function _GetOpenWindowJS(&$oPopupTableConf)
         return $js;
     }
 
+    private function getTargetListRestriction(): array
+    {
+        $restrictionExpression = $this->oDefinition->GetFieldtypeConfigKey('restriction');
+        if (null === $restrictionExpression) {
+            return [];
+        }
+        if (false === \strpos($restrictionExpression, '=')) {
+            return [];
+        }
+        $equalPosition = \mb_strpos($restrictionExpression, '=');
+        $restrictionField = \trim(\mb_substr($restrictionExpression, 0, $equalPosition));
+        $restrictionValue = \mb_substr($restrictionExpression, $equalPosition + 1);
+
+        if (true === $this->isFieldToken($restrictionValue)) {
+            $restrictionValueReplaced = $this->getFieldValueFromFieldToken($restrictionValue);
+            if (null === $restrictionValueReplaced) {
```

Comment:
Can such a value (in the target table) be correctly empty or null and thus falsely trigger this error?

Reply:
we return null only if the field is not set or is null. Since we currently do not support null value fields, I think the code can safely stay as it is.